src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,45 @@
+.clock {
+  position: relative;
```

Comment:
do not set position or margins to the block

Reply:
use mixes

Reply:
Done